modules/core/src/v2/internal/seedValidator.ts

```diff
@@ -0,0 +1,71 @@
+import { isValidSeed as AlgoIsValidSeed } from 'algosdk';
+import * as stellar from 'stellar-sdk';
+import { Ed25519PrivateKey } from '@hashgraph/sdk';
```

Comment:
Yikes.

Reply:
ok, removed it and moved this into account-lib